src/app/Styles/talking.css

```diff
@@ -248,6 +248,7 @@
 
 .chat-part .chatinput-extented {
     width: 100%;
+    border-radius: 6px 6px 6px 6px;
```

Comment:
`border-radius: 6px;` should have the same effect.

Reply:
Good idea!